ruma-api-macros/src/api.rs

```diff
@@ -269,8 +269,8 @@ impl ToTokens for Api {
                     *http_request.uri_mut() = ruma_api::exports::http::uri::Builder::new()
                         .path_and_query(path_and_query.as_str())
                         .build()
-                        // The only way this can fail is if the path given in the API definition is
-                        // invalid. It is okay to panic in that case.
+                        // The ruma_api! macro guards against invalid path input but, if there are
+                        // invalid (non ASCII) bytes in the fields with the query attribute this will panic.
```

Comment:
This comment makes it sound like this is *not* guaranteed to succeed.

Reply:
The unwrap can fail still, if the query string had `vec![("abc", "µ / ° / § / €")]` it would fail? Or is it impossible for that to happen because of percent encoding?

Reply:
Hm, I don't actually know. I now understand what this comment is talking about though :D

Just put the comma before the "but" and I'm happy for now :)